mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -132,10 +133,12 @@ def __call__(self, datasets):
         if self.featsel != 1.0:
             # computing feature scores from the data
             feature_scores = compute_feature_scores(datasets, self.exclude_from_model)
-            if self.featsel < 1.0:
-                fselector = FractionTailSelector(self.featsel, tail='upper', mode='select', sort=False)
-            else:
-                fselector = FixedNElementTailSelector(np.floor(self.featsel), tail='upper', mode='select', sort=False)
+            nfeatures_sel = nfeatures  # default
+            if self.featsel < 1.0 and np.floor(self.featsel * nfeatures):
```

Comment:
Why the floor? If needed, can we use explicitly `np.floor(self.featsel * nfeatures)`?